heareval/task_embeddings.py

```diff
@@ -22,102 +22,117 @@
     https://github.com/neuralaudio/hear2021-eval-kit/issues/49
 """
 
-import csv
 import glob
+import json
 import os.path
 from importlib import import_module
-from typing import Any, Dict, Tuple
+from typing import Any, Dict, Optional, Tuple
 
 import numpy as np
+import pandas as pd
 import soundfile as sf
 import torch
 from torch.utils.data import DataLoader, Dataset
 from tqdm.auto import tqdm
 
 # This could instead be something from the participants
-EMBEDDING_PIP = "heareval.model.baseline"
+# TODO: Command line or config?
```

Comment:
Ya let's do this similar to hear validator? We can update to use click -- https://github.com/neuralaudio/hear-validator/blob/7d28d357620fff4c4a533e232a1079a1db785a49/hearvalidator/validate.py#L395 ?